odk1-src/central-forms.rst

```diff
@@ -39,34 +44,53 @@ You can either click on the :guilabel:`choose one` button to browse for your :fi
 
-   .. image:: /img/central-forms/checklist.png
+If your Draft requires Media Files, there will be a checklist step asking you to upload them, and a Media Files tab at the top of the page. See the next section :ref:`Forms With Attachments <central-forms-attachments>` for more information about uploading and managing attachments.
 
-The documentation on this page is a more detailed introductory explanation of form management in ODK Central, but the checklist you find on the Overview page is tailored to the current status of your form and your project and is a great place to look when you aren't sure what to do next.
+The :guilabel:`Testing` Draft tab shows test submissions that have been made to the Draft, and instructions for doing so:
 
-In the future, look forward to seeing even more useful information at-a-glance on this page.
+   .. image:: /img/central-forms/testing.png
+
+At the top of the page are instructions and a QR Code which will configure a mobile device to submit to the Draft Form. For help configuring a mobile device, please see :doc:`importing settings into Collect <collect-import-export>`. The table below these instructions contains any test submissions that have been made to the current Draft. For help with this table or exporting test data, please see :doc:`Form Submissions in Central <central-submissions>`.
 
 .. _central-forms-attachments:
 
 Forms With Attachments
 ----------------------
 
-If your uploaded form references any external files (images, audio, or video included as part of your question prompts, or data lookup files used to populate selection lists), Central will see this and open up some additional displays and controls you will need to provide those external files:
+If your Form Draft references any external files (images, audio, or video included as part of your question prompts, or data lookup files used to populate selection lists), Central will see this and open up some additional displays and controls you will need to provide those external files:
 
    .. image:: /img/central-forms/attachments-overview.png
 
-If you see this extra **Upload Form Media Files** checklist step and **Media Files** tab at the top of your form overview, then Central believes you need to upload some files associated with this form. If the checklist step has been checked off, then you've already completed this task: great work! Otherwise, click on the :menuselection:`--> Media Files` tab at the top to see what files you'll need to provide.
+If you see this extra **Upload Form Media Files** checklist step and **Media Files** tab at the top of your Form Draft checklist, then Central believes you need to upload some files associated with this form. If the checklist step has been checked off, then you've already completed this task: great work! Otherwise, click on the :menuselection:`--> Media Files` tab at the top to see what files you'll need to provide.
```

Comment:
Not sure we need the `-->` here.

Reply:
eh. it was there before. i'll just leave it there.